terraform/helm/aptos-node/templates/_helpers.tpl

```diff
@@ -46,6 +46,20 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
 
+{{/*
+Multicluster labels
+*/}}
+{{- define "aptos-validator.multiclusterLabels" -}}
+{{- $ctx := index $ 0 -}}
```

Comment:
maybe include some comments on how this works. Took me a while to figure out the `index $ 0` syntax was for fetching the function params, like doing something like: `"aptos-validator.multiclusterLabels" (tuple $ $i)`